client_internals_it_test.go

```diff
@@ -542,3 +543,176 @@ func ensureClient(config hz.Config) *hz.Client {
 	}
 	panic("the client could not connect to the cluster in 60 seconds.")
 }
+
+type MutualAuthenticationStatus string
+
+const (
+	OPTIONAL MutualAuthenticationStatus = "OPTIONAL"
+	REQUIRED MutualAuthenticationStatus = "REQUIRED"
+)
+
+func TestMutualAuthentication(t *testing.T) {
```

Comment:
Could you rename the test to `mutualAuthenticationTest` and add it to `TestClientInternal`?

Reply:
Btw, why is this test in `client_internals_it_test.go`? As far as I see it doesn't require anything from `ClientInternals`.

Reply:
Moved to another source, [c6481ec](https://github.com/hazelcast/hazelcast-go-client/pull/859/commits/c6481ec04328cf044ab3bec8e31bd712bd663bd2).